cmd/virt-api/virt-api.go

```diff
@@ -136,10 +138,35 @@ func (app *virtAPIApp) Run() {
 		},
 	}
 	swagger.InstallSwaggerService(config)
+	openapiConf := restfulspec.Config{
+		WebServices:    []*restful.WebService{ws},
```

Comment:
Can you do the same at https://github.com/kubevirt/kubevirt/pull/494/files#diff-6d08dabd2f2152335fa760fef7fd4cc5R127?

Reply:
Changing that line makes the `http://192.168.200.2:8183/swaggerapi/apis/kubevirt.io/v1alpha1` to return `No APIDefinition Found`

Reply:
On other hand you are right, I have also swagger entry point included in openapi specification.
And there are errors like:
```
Semantic error at paths./swaggerapi/{a}/{b}.get.operationId
operationIds must be unique
```

Reply:
It would be ok for me, if you just drop the swagger spec.

Reply:
OK, I will drop it.

Reply:
@rmohr It is not so easy as I expected, the swagger is used on several places. I will also have to perform changes in virt-api Dockerfile, in haproxy, and generated types ...

I am going to give a deeper dive to it, and I will let you know.

Reply:
I don't think that you have to change gnerated types. We generate special "swagger" files, to enrich the documentation, but they should also be used and picked up for openapi.

Reply:
I will send what I have, I will appreciate your input!